region/maridia/inner-green.json

```diff
@@ -461,7 +461,7 @@
           "sparking": {
             "runways": [
               {
-                "length": 7,
+                "length": 6,
```

Comment:
? Looks like 7 to me

Reply:
There's an enemy there and I wanted to make sure that it doesn't interfer if coming in. Also an extra runway for 1 more tile felt pointless.

Reply:
Well, that tile could be the one that makes a different in some cases. I don't think having two runways would be bad.